### PR TITLE
MOTECH-2065 Create a document that lists MOTECH's current capabilities

### PR DESCRIPTION
**Pull request by craigappl**

### CODE REVIEW

File: docs/source/current_capabilities.rst [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    - Import historical forms
+    - Connect to Commcare's data, fixture and location APIs to supplement tasks
+- DHIS2 one-way push integration
+    - Configure a single connection to a DHIS2 server
+    - Create tracked entity instances
+    - Enroll tracked entities in DHIS2 programs
+    - Push program stage events
+    - Send aggregate data values and value sets
+- SMS Integration
+    - Send and receive SMS from MOTECH through integration with an SMS aggregater with predefined configurations for popular systems like Clickatell, Plivo, Twilio and Voto
```
Comment (pgesek) [inline docs/source/current_capabilities.rst]: aggregator?

File: docs/source/current_capabilities.rst [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+- IVR Integration
+    - Initiate outgoing calls from MOTECH by integrating with a mobile network operator
```
Comment (pgesek) [inline docs/source/current_capabilities.rst]: We can also handle inbound calls

File: docs/source/current_capabilities.rst [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+- Send emails through the UI
+- View email logs
+
+Scheduler
+"""""""""
+- View scheduler jobs
+
+Tasks
+"""""
+- Create and manage tasks (note: one task can perform one action)
```
Comment (pgesek) [inline docs/source/current_capabilities.rst]: a task can have multiple actions

**Comment by craigappl** [comment]
@pgesek just got great feedback from @nickdotreid and will rearrange this document tomorrow.

**Comment by craigappl** [comment]
Hi Guys,

We realized that the tab-by-tab description is better for the UI walk through and moved them there. I also tried to make better categories of current capabilities for users who have never used MOTECH.

**Comment by craigappl** [comment]
@pgesek  Can you take another look at this?

**Pull request merged**